vortex-jni/src/array.rs

```diff
@@ -53,6 +56,105 @@ pub extern "system" fn Java_dev_vortex_jni_NativeArrayMethods_free(
     drop(unsafe { NativeArray::from_raw(array_ptr) });
 }
 
+#[unsafe(no_mangle)]
+pub extern "system" fn Java_dev_vortex_jni_NativeArrayMethods_exportToArrow<'local>(
+    mut env: JNIEnv<'local>,
+    _class: JClass,
+    array_ptr: jlong,
+    arrow_schema_ptr: JLongArray<'local>,
+    arrow_array_ptr: JLongArray<'local>,
+) {
+    // Return the arrow array here instead...I think?
```

Comment:
We can't do that, right? Because there is no single "arrow array with schema" at the C-level.